### ISSUE TITLE
Help to test MeteoalarmCard 2.0

### ISSUE THREAD

**Issue by MrBartusek** [MrBartusek/MeteoalarmCard#88]
Hey! I have just released first release candidate for version 2.0. It's the biggest update that this project has received. You can read full release notes [here](https://github.com/MrBartusek/MeteoalarmCard/releases). The full release will be coming in a couple of weeks, depending on how many issues there will be. You can download the new version and report any issues, feature requests or thoughts in the [issue tracker](https://github.com/MrBartusek/MeteoalarmCard/issues).

This release adds a bunch of new translation keys and you will see a lot of missing strings in this version. The current translation progress is as follows:
```diff
+ English - 100% translated
+ Polish - 100% translated
- Czech -  57% translated
- Spanish - 57% translated
- French - 57% translated
- German - 55% translated
- Estonian - 55% translated
- Croatian - 55% translated
- Slovak - 55% translated
- Swedish - 50% translated
- Dutch - 40% translated
- Italian- 29% translated
```

You can help to add missing keys in existing languages or add a new language - see [CONTRIBUTING.md](https://github.com/MrBartusek/MeteoalarmCard/blob/master/CONTRIBUTING.md)!

## How to install beta version 

You can download beta version using HACS

1. Re-donwload MeteoalarmCard
![image](https://user-images.githubusercontent.com/23432278/174665146-5ae7e9e8-c2bd-452e-a75f-1d14feac78ad.png)

2. Check *Show beta versions*
![image](https://user-images.githubusercontent.com/23432278/174665333-258e9d08-4e32-4992-b4bf-9e7fc22ce5bf.png)

3. Select newest release candidate
![image](https://user-images.githubusercontent.com/23432278/174665387-285149f8-02e0-4d11-b14c-da0ec352815c.png)

4. Donwload new version
![image](https://user-images.githubusercontent.com/23432278/174665452-f64417f7-ee88-4055-afe1-2ce83ef854ac.png)

**Comment by Gavin-A92** [comment]
Hi...I've installed the RC for v2. Reconfigured and all works fine I think with the exception of the `hide_when_no_warning:` flag, even when set to True the card is appearing even without an active warning.....
Think I have the new config correct - see below.

Running latest version (2022.6.6) of HA if that's relevant. Let me know if you need anything else 

```
type: custom:meteoalarm-card
hide_when_no_warning: true
override_headline: false
entities:
  - entity: binary_sensor.meteoalarm_home
integration: meteoalarm
```

**Comment by NachtaktiverHalbaffe** [comment]
I also have the bug where the ``hide_when_no_warning'' does nothing. I could also maybe contribute to the german translation. Do I just need to insert the translations into [MeteoalarmCard](https://github.com/MrBartusek/MeteoalarmCard)/[src](https://github.com/MrBartusek/MeteoalarmCard/tree/master/src)/[localize](https://github.com/MrBartusek/MeteoalarmCard/tree/master/src/localize)/[languages](https://github.com/MrBartusek/MeteoalarmCard/tree/master/src/localize/languages)/de.json on the 2.0 branch and then create a PR?

**Comment by MrBartusek** [comment]
Yes, 2.0 is merged to master

**Comment by NachtaktiverHalbaffe** [comment]
I am using Nina and DWD integration. On a wide screen/web it shows the actual warning. When I shrink it down to mobile format i.e. look on the mobile add, then the NINA integration just shows "orange warning" as a warning and DWD the short term for the corresponding warning. Is this intended?

Because NINA uses the DWD data in the background for those types of warning (weather), both should/could have the same short term warning titles for the weather events

**Comment by MrBartusek** [comment]
Hey, @NachtaktiverHalbaffe card always generates a couple of headlines and uses them depending on the current screen width so it can be different depending on the used device.

![scaling](https://user-images.githubusercontent.com/23432278/175565153-e8458bb9-699d-4e81-9fff-69700ed4e01a.gif)

You may see a difference in DWD and NINA alerts since NINA doesn't provide event type (used for the icon) and headline while DWD gives both of them.

**Comment by MrBartusek** [comment]
@Gavin-A92 @NachtaktiverHalbaffe yes I can confirm that `hide_when_no_warning` is broken and this will be fixed.

**Comment by MrBartusek** [comment]
Hey! There is a new beta version https://github.com/MrBartusek/MeteoalarmCard/releases/tag/v2.0.0-rc.2 which fixes `ide_when_no_warning`

**Comment by Gavin-A92** [comment]
Have installed the new beta and fixes the `idle_when_no_warning` for me so all looks good....thanks for your work on this

**Comment by MrBartusek** [comment]
Hey! The estimated time of the 2.0.0 release is 7th July.
